demos/guru_scripts/docker/tutorial/3.x/gsql101/README.md

```diff
@@ -0,0 +1,27 @@
+Contents
+==============
+This GSQL 101 tutorial contains 
+
+- friendship.csv: the edge data
+- person.csv: the vertex data
+- gsql101.gsql: the gsql script to setup the schema, load the data, and stored procedures. 
+
+
+How To Run
+===============
+gsql gsql101.gsql
+
+
+Documents
+==============
+https://docs.tigergraph.com/intro/gsql-101
+
+
+Support
+===============
+If you like the tutorial and want to explore more, join the GSQL developer community at 
+
+https://groups.google.com/a/opengsql.org/forum/?hl=en#!forum/gsql-users
```

Comment:
this google group has expired. We moved to this
https://community.tigergraph.com/

So, please check all such links, and update them to the above community link.